app/src/schema.json

```diff
@@ -0,0 +1,407 @@
+{
+  "title": "NMIND checklist",
+  "description": "A good description",
+  "type": "object",
+  "properties": {
+    "name": {
+      "title": "Software name",
+      "type": "string"
+    },
+    "urls": {
+      "title": "Links",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "url": {
+            "title": "Link",
+            "prompt": "Link",
+            "type": "string"
+          },
+          "url_type": {
+            "title": "Link type",
+            "prompt": "type",
+            "type": "string",
+            "enum": [
+              "Landing page",
+              "Source code",
+              "Documentation"
+            ],
+            "default": "Landing page"
+          }
+        },
+        "required": [
+          "url",
+          "url_type"
+        ],
+        "additionalProperties": false
+      },
+      "default": []
+    },
+    "documentation": {
+      "type": "object",
+      "properties": {
+        "bronze": {
+          "type": "object",
+          "properties": {
+            "1": {
+              "type": "boolean",
```

Comment:
Exactly, it would look something like this:

```JSON
"2": {
  "type": "object",
  "title": "Documentation is up to date with version of software",
  "properties": {
    "value": {
      "type": "boolean",
      "default": false
    }
    "comment": {
      "type": "string",
      "default": ""
    }
  }
},
```

To avoid repetition this could be [`$def`-ined once and then used for every item](https://json-schema.org/understanding-json-schema/structuring#dollardefs).

Reply:
https://github.com/nmind/standards-checklist/issues/33